bemani/common/card.py

```diff
@@ -200,9 +200,9 @@ class CardCipher:
 
     @staticmethod
     def __type_from_cardid(cardid: str) -> int:
-        if cardid[:2].upper() == 'E0':
```

Comment:
What caused you to change this? I'm not sure at all which is correct, but I hesitate to make changes to long-baked code that's known working.

Reply:
There were valid Felica ids that broke the server and I couldn't actually figure out what all the possible valid ones were so i just let it accept whatever.

Reply:
Hmm. I think that this isn't harmful. The only real thing could be bootleg cards that were not "E004" but started with "E0" and used the v1 cypher for the back printing? Those would render on the UI as the v2 cypher now, wouldn't they? Do you know what prefix the felica ids were? Shouldn't we whitelist those/add them to the v2 cypher instead of tightening the v1?